### ISSUE TITLE
Getting undefined method `mattr_reader'

### ISSUE THREAD

**Issue by MikeBlyth** [quirkey/gembox#13]
I see a hint that this occurred earlier, but don't know how to approach it. I just installed gembox 0.2.2 and this is the error I get when I try to run it. Is it a problem with will-paginate rather than gembox?

```
/home/mike/.rvm/gems/ruby-1.9.2-p0/gems/will_paginate-2.3.15/lib/will_paginate view_helpers.rb:37:in `<module:ViewHelpers>': undefined method `mattr_reader' for WillPaginate::ViewHelpers:Module (NoMethodError)
from /home/mike/.rvm/gems/ruby-1.9.2-p0/gems/will_paginate-2.3.15/lib/will_paginate/view_helpers.rb:22:in `<module:WillPaginate>'
from /home/mike/.rvm/gems/ruby-1.9.2-p0/gems/will_paginate-2.3.15/lib/will_paginate/view_helpers.rb:3:in `<top (required)>'
from <internal:lib/rubygems/custom_require>:29:in `require'
from <internal:lib/rubygems/custom_require>:29:in `require'
from /home/mike/.rvm/gems/ruby-1.9.2-p0/gems/gembox-0.2.2/lib/gembox.rb:10:in `<top (required)>'
from <internal:lib/rubygems/custom_require>:29:in `require'
from <internal:lib/rubygems/custom_require>:29:in `require'
from /home/mike/.rvm/gems/ruby-1.9.2-p0/gems/gembox-0.2.2/bin/gembox:6:in `<top (required)>'
from /home/mike/.rvm/gems/ruby-1.9.2-p0/bin/gembox:19:in `load'
from /home/mike/.rvm/gems/ruby-1.9.2-p0/bin/gembox:19:in `<main>'
```

**Comment by quirkey** [comment]
Moving to jeweler. Fixed will-paginate issue. Closed by 8e4b9ce7fec703365829186e518aeb87df746d7f

**Comment by ricpruss** [comment]
This is still happening for me.
